og_ui/og_ui.admin.inc

```diff
@@ -998,9 +998,29 @@ function theme_og_ui_admin_permissions($variables) {
  * Allow site admin to add or remove group fields from fieldable entities.
  */
 function og_ui_field_settings($form, &$form_state) {
+
+
+
   $form = array();
   $options = array();
 
+  $help = '<p>' . t('Organic Groups provides a number of fields that can be added to bundles (such as nodes or entities). Examples:') . '</p>';
+  $help .= '<ul>';
+  $help .= '<li>' . t('Add a "group" field to a content type to configure it as a group.') . '</li>';
+  $help .= '<li>' . t('Add a "group audience" field a content type to assign content to a group.') . '</li>';
```

Comment:
Even if there are already plenty, is here a `to` missing? Should probably be:

> Add a "group audience" field **to** a content type to assign content to a group.

Reply:
Fixed.